src/components/chart/specific-chart-options.js

```diff
@@ -1,5 +1,9 @@
 class SpecificChartOptions {
-    constructor(theme, type) {
+    constructor(theme, type, useStackedLayout, config) {
```

Comment:
@precious-onyenaucheya-ons I think this file is getting less useful and more of a sprawl now. I'd like to refactor it so that we start to have all the specific options just in `bar-chart.js`, `line-chart.js` etc. I'm going to look into this as part of this PR if you're happy with that appraoch.